build.gradle.kts

```diff
@@ -39,23 +42,22 @@ kotlin {
         val commonNonJvmMain by creating {
             dependsOn(commonMain)
         }
-        nativeTargets.forEach { nativeTarget ->
+        nonJvmTargets.forEach { nativeTarget ->
```

Comment:
these wrappers with `nonJvmTargets` are peter's complication
I duplicate code a little bit, but have a more plain and readable structure of project in ktoml, also no problems with imports in idea